test/helpers/kubectl.go

```diff
@@ -68,7 +68,7 @@ var (
 	defaultHelmOptions = map[string]string{
 		"global.registry":               "k8s1:5000/cilium",
 		"agent.image":                   "cilium-dev",
-		"agent.tag":                     "latest",
+		"global.tag":                    "latest",
 		"operator.image":                "operator",
 		"operator.tag":                  "latest",
```

Comment:
Nice catch! Shouldn't we get rid of `operator.tag` as the operator Chart is using the `global.tag` to define the image tag?

Reply:
whoah, didn't see this one. Will add this to this PR

Reply:
I think we should revisit all the options.